source/ShipInfoDisplay.cpp

```diff
@@ -37,16 +38,16 @@ using namespace std;
 
 ShipInfoDisplay::ShipInfoDisplay(const Ship &ship, const Depreciation &depreciation, int day)
 {
-	Update(ship, depreciation, day);
+	Update(ship, depreciation, day, true);
```

Comment:
Why is this one true? We dont know if we're im the shipyard or not mb that info should be passed along?
It doesnt truly matter because it'll be refreshed right afterwards but still.

Reply:
Maybe it should be removed entirely and just default to false? I'm not sure how I would pass along the sale value here, or even if its worth it.

Reply:
Best let it default to false and not show things we dont want to if there is an issue